packages/flutter/lib/src/material/bottom_sheet.dart

```diff
@@ -112,6 +116,12 @@ class BottomSheet extends StatefulWidget {
   /// Default is true.
   final bool enableDrag;
 
+  /// Specifies whether a drag handle is shown when [enableDrag] is true.
+  ///
+  /// When null (the default), the behavior is determined by
+  /// [BottomSheetThemeData.hasDragHandle].
```

Comment:
This BottomSheetThemeData property doesn't seem to exist.  Perhaps it should

Reply:
Good catch, thanks! I think in my comment at https://github.com/flutter/flutter/pull/120855#discussion_r1125199064 I thought I was looking at the theme, but in fact that was on the route.